src/_pytest/junitxml.py

```diff
@@ -664,7 +663,9 @@ def pytest_sessionfinish(self) -> None:
                 skipped=str(self.stats["skipped"]),
                 tests=str(numtests),
                 time=f"{suite_time_delta:.3f}",
-                timestamp=datetime.fromtimestamp(self.suite_start_time, timezone.utc)
+                timestamp=datetime.fromtimestamp(
+                    self.suite_start_instant.interval()[0], timezone.utc
```

Comment:
Instant should provide an as_datetime returning the utc datetime

Reply:
Do you mean a `interval_as_utc_datetime()` method?

Reply:
No need for interval in the method name but yes

Maybe just as_datetime() always returning utc

Reply:
You mean two methods, `start_as_datetime` and `end_as_datetime`?

Reply:
no - a instant is only the instant - there is no start/end  - - also this code asks for the start time from the "interval" not the full time

Reply:
Not sure I understand, note that the original code was:

```python
timestamp=datetime.fromtimestamp(self.suite_start_time, timezone.utc)
```

So it was definitely using the timestamp related to the start time of the test suite only, and `inverval()[0]` corresponds to `time.time()` at the creation of the `Instant`.

Reply:
The instant itself is the start 

So instant.as_datetime() should be the api

Reply:
I see, but take a look at my implementation, I went with a slightly different route but I think it is better now and provides an API similar to what you asked.